ui/app/components/app/alerts/alerts.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import UnconnectedAccountAlert from './unconnected-account-alert'
+import { alertIsOpen as unconnectedAccountAlertIsOpen } from '../../../ducks/alerts/unconnected-account'
+
+const Alerts = () => {
+  const isOpen = useSelector(unconnectedAccountAlertIsOpen)
```

Comment:
Nit: this is a poor choice in variable name, since I'll be adding similar variables for each other alert as well (they can't all be called `isOpen`).